### PR TITLE
Add unit test

### PR DESCRIPTION
**Pull request by lbengayed**

### CODE REVIEW

File: services/src/test/java/org/exoplatform/task/service/TaskServiceTest.java [hunk 1]
```diff
@@ -252,6 +295,23 @@ public void testUpdateTaskDueDate() throws ParameterEntityException, EntityNotFo
...
+    TaskDto task = taskService.getTask(TestUtils.EXISTING_TASK_ID);
+    Status status =new Status();
+    long[] ids = new long[1];
+    task.setStatus(status);
+    task.setStatus(status);
+    // when(taskStorage.updateTaskOrder(any(),status,ids)).thenReturn(task);
+    taskService.updateTaskOrder(TestUtils.EXISTING_TASK_ID,status,ids);
+    //verify(taskHandler, times(2)).updateTaskOrder(taskCaptor.capture().getId(),status,ids);
+
+    // assertEquals(newTitle, taskCaptor.getValue().getTitle());
```
Comment (mkrout) [inline services/src/test/java/org/exoplatform/task/service/TaskServiceTest.java:311]: ?

File: services/src/test/java/org/exoplatform/task/service/TaskServiceTest.java [hunk 2]
```diff
@@ -221,6 +214,56 @@ public void testUpdateTaskCoworker() throws ParameterEntityException, EntityNotF
...
+    Set<String> newWatcher = new HashSet<String>();
+    for (String v : newWatchers) {
+      newWatcher.add(v);
+    }
+    assertEquals(newWatcher, taskCaptor.getValue().getWatcher());
+   /* newWatchers.remove("Tib");
+    task.setWatcher(newWatchers);
+    when(daoHandler.getTaskHandler().update(any())).thenReturn(taskStorage.toEntity(task));
+    taskService.deleteWatcherOfTask("Tib",task);
+*/
```
Comment (mkrout) [inline services/src/test/java/org/exoplatform/task/service/TaskServiceTest.java:263]: Please uncomment or remove commented code

**Review by mkrout — CHANGES_REQUESTED** [review]

File: services/src/test/java/org/exoplatform/task/service/TaskServiceTest.java (hunk 1 above)
Comment (lbengayed) [reply services/src/test/java/org/exoplatform/task/service/TaskServiceTest.java]: done!

File: services/src/test/java/org/exoplatform/task/service/TaskServiceTest.java (hunk 2 above)
Comment (lbengayed) [reply services/src/test/java/org/exoplatform/task/service/TaskServiceTest.java]: done

**Review by mkrout — APPROVED** [review]
LGTM

**Pull request merged**